moped-editor/src/auth/user.js

```diff
@@ -95,11 +95,25 @@ export const UserProvider = ({ children }) => {
   return <UserContext.Provider value={values}>{children}</UserContext.Provider>;
 };
 
+export const getRandomColor = () => {
+  const key = "atd_moped_user_color";
+  if (localStorage.getItem(key) === null) {
+    const randomInt = Math.floor(Math.random() * Math.floor(3));
+    const colors = ["#43a047", "#3949ab", "#fb8c00", "#e53935"];
```

Comment:
should we use colors based on our Material Theme? Or at least store these hex codes as color variables in a constants file for re-use?

Reply:
I took those from our material UI theme, I can re-work these as variables in a config file

Reply:
Yeah, I think we should either use hexcode from or add to the `theme.palette` object or even make a new palette.js or color.js file in the `/theme` directory.